### ISSUE TITLE
Airports lost timezones

### ISSUE THREAD

**Issue by vandervecken** [jpatokal/openflights#1023]
Hi,
I last took a cut of this data in November 2016. I took a new cut yesterday. The following airports have lost their correct timezone data in that time. Some of them are pretty major:

airport_id | Name | IATA | New Timezone | Old Timezone
-------------------------------------------------------------
9904 | Richmond Airport | RCM | \N | Australia/Sydney
10116 | Bunbury Airport | BUY | \N | America/New_York
10122 | Elkhart Municipal Airport | EKI | \N | America/New_York
11051 | Hamad International Airport | DOH | \N | Asia/Qatar
11900 | King Salman Abdulaziz Airport | DWD | \N | Asia/Riyadh
12087 | Rajiv Gandhi International Airport | HYD | \N | Asia/Calcutta
13183 | Bonn-Hangelar Airport | BNJ | \N | Europe/Berlin
13696 | Istanbul Airport | IST | \N | Europe/Istanbul
13711 | Lutselk'e Airport | YSG | \N | America/Edmonton
13754 | La Araucanía Airport | ZCO | \N | America/Santiago
13760 | New Islamabad International Airport | ISB | \N | Asia/Karachi
14102 | Laguindingan Airport | CGY | \N | Asia/Manila
14109 | Desierto de Atacama Airport | CPO | \N | America/Santiago

**Comment by bmomjian** [comment]
Yes, I can confirm this issue is still a problem for Hamad International Airport / DOH.  This issue is obviously not moving forward.  Would a submitted pull request help advance this?
